.github/workflows/build.yaml

```diff
@@ -23,28 +23,17 @@ on:
         type: string
 
 permissions:
-  packages: read
+  packages: write
```

Comment:
I would need to refresh my memory on how permissions & reusable workflows work to know for sure.. but can we leave this to `read` and only plumb through the write permission on the resusable workflow call? That is, can we do something like
```yaml
 build-base:
    uses: .github/workflows/build_base.yaml
    permissions:
       packages: write
       contents: read
```

Reply:
Good call. Finally got that to work. Had tried a couple other options before and didn't get the right combination it seems.  Addressed: 495d6eab30fcde